docs/faq/product-faq.md

```diff
@@ -0,0 +1,68 @@
+---
+id: Product FAQs
+title: Product FAQs
+description: All FAQs related to Product
+---
+
+## Product FAQs
```

Comment:
No need to have this here, the `title` from above will already be at the top of the page so this would be a duplicate. Still best to keep the questions as `###` though, `##` would be too large